src/Cursors.ts

```diff
@@ -137,6 +137,31 @@ export default class Cursors extends EventEmitter<CursorsEventMap> {
     const channel = this.getChannel();
     return await this.cursorHistory.getLastCursorUpdate(channel, this.options.paginationLimit);
   }
+
+  async getSelf(): Promise<CursorUpdate | undefined> {
+    const allCursors = await this.getAll();
+    const self = this.space.getSelf();
+    if (!self || !allCursors) {
```

Comment:
`allCursors` will never be falsy - it always returns an object (so we just need the `!self` here)